modules/microshift-nw-ipv6-dual-stack-migrating-config.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-using-config-tools.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-nw-ipv6-dual-stack-migrating-config_{context}"]
+= Migrating a {microshift-short} cluster to IPv6 dual-stack networking
+
+You can convert a single-stack cluster to dual-stack cluster networking that supports IPv4 and IPv6 address families by setting two entries in the service and cluster network parameters in the {microshift-short} configuration file.
+
+* The first IP family in the configuration is the primary IP stack in the cluster.
+* {microshift-short} system pods and services are automatically updated upon {microshift-short} restart.
+* After the cluster is migrated to dual-stack networking and has restarted, enable workload pods and services for dual-stack networking by restarting them.
+
+[IMPORTANT]
+====
+The OVN-Kubernetes network plugin requires that both IPv4 and IPv6 default routes be on the same network device. IPv4 and IPv6 default routes on separate network devices is not supported.
+====
+
+[IMPORTANT]
+====
+When using dual-stack networking where IPv6 is required, you cannot use IPv4-mapped IPv6 addresses, such as `::FFFF:198.51.100.1`.
+====
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* Your cluster uses the OVN-Kubernetes network plugin.
+* The host has both IPv4 and IPv6 addresses and routes, including a default for each.
+* The host has at least two L3 networks, IPv4 and IPv6.
+
+.Procedure
+
+. If you have not done so, make a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`.
+
+. Keep the new {microshift-short} `config.yaml` in the `/etc/microshift/` directory. Your `config.yaml` file is read every time the {microshift-short} service starts.
++
+[NOTE]
+====
+After you create it, the `config.yaml` file takes precedence over built-in settings.
+====
+
+. Add IPv6 configurations to the `network` section of the {microshift-short} YAML with your desired valid values:
++
+[WARNING]
+====
+You must keep the same first entry across restarts and migrations. This is true for any migration, single-to-dual stack, or dual-to-single stack. A complete wipe of the etcd database is required if a change to the first entry is needed. This might result in application data loss and is not supported.
```

Comment:
```suggestion
You must keep the same first entry across restarts and migrations. This is true for any migration: single-to-dual stack, or dual-to-single stack. A complete wipe of the etcd database is required if a change to the first entry is needed. This might result in application data loss and is not supported.
```